src/transformers/dynamic_module_utils.py

```diff
@@ -185,35 +185,20 @@ def check_imports(filename: Union[str, os.PathLike]) -> List[str]:
     return get_relative_imports(filename)
 
 
-def get_class_in_module(repo_id: str, class_name: str, module_path: Union[str, os.PathLike]) -> typing.Type:
+def get_class_in_module(class_name: str, module_path: Union[str, os.PathLike]) -> typing.Type:
     """
     Import a module on the cache directory for modules and extract a class from it.
 
     Args:
-        repo_id (`str`): The repo containing the module. Used for path manipulation.
         class_name (`str`): The name of the class to import.
         module_path (`str` or `os.PathLike`): The path to the module to import.
```

Comment:
module_path now expects the file extension too now. Maybe that should be reflected here too? @cebtenzzre